lua/lvim/core/which-key.lua

```diff
@@ -244,6 +244,7 @@ M.config = function()
         c = { "<cmd>Telescope colorscheme<cr>", "Colorscheme" },
         f = { "<cmd>Telescope find_files<cr>", "Find File" },
         h = { "<cmd>Telescope help_tags<cr>", "Find Help" },
+        H = { "<cmd>Telescope highlights<cr>", "Find highlight groups" },
```

Comment:
```suggestion
```

Reply:
Was this removed from master? Or is it somewhere else?